copc/lidar-las-to-copc.ipynb

```diff
@@ -0,0 +1,707 @@
+{
```

Comment:
<p>It's hard for me to understand what you are trying to show here. Are we just trying to show what is in the metadata or are we trying to show the values of the stats in the metadata? </p><p> </p><p>If the latter maybe something like this?</p><p> </p>

<pre class="ql-syntax" spellcheck="false">valid_pipe = pdal.Reader.las(filename=las_filename) | pdal.Filter.stats()
valid_pipe.execute()
valid_pipe.metadata["metadata"]["filter.stats"]

</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cloudnativegeo/cloud-optimized-geospatial-formats-guide/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Or maybe you want to show what is within the <code>"statistics"</code></p><p> </p><p><code></code><code></p><p>valid_pipe.metadata["metadata"]["filters.stats"]["statistic"] </p><p></code><code></code></p>

Reply:
<p>I agree. I was trying to show the dictionary keys and then the user to decide what to show instead of hard coding but it makes sense to keep it simple. Thanks. </p>